app/src/main/java/mozilla/lockbox/view/EditItemFragment.kt

```diff
@@ -159,6 +176,10 @@ class EditItemFragment : BackableFragment(), EditItemDetailView {
         }
     }
 
+    override fun setTextSelectionOnPasswordToggle() {
```

Comment:
I really don't understand the intent of this method.

Reply:
good point. this method takes care of this: 
> Toggling between hiding and showing a password in the field moves the text cursor between the start and the end of the string; it should stay at the end of the string